scss/_base_code.scss

```diff
@@ -0,0 +1,28 @@
+// Base code styles
+@mixin vf-b-code {
+  code,
+  samp,
+  kbd {
+    font-family: $font-monospace;
+    text-align: left;
+  }
+
+  pre,
+  code {
+    text-align: left;
+    white-space: pre-line;
+    word-spacing: normal;
+    word-wrap: break-word;
+    tab-size: 4;
+    hyphens: none;
+    direction: ltr;
+  }
+
+  pre {
+    background: $color-dark;
+    color: $color-light;
+    border: 1px solid $color-mid-grey;
+    padding: 1rem;
```

Comment:
Grid guttering with be reduced to 16px, box pattern is being deprecated and card pattern will have 1rem padding also.